src/app/dropdown/dropdown.component.ts

```diff
@@ -1,27 +1,43 @@
-import { Component, ElementRef, OnInit } from '@angular/core';
+import { Component, OnInit, EventEmitter, Input, Output, SimpleChange } from '@angular/core';
+
 
 @Component({
   selector: 'app-dropdown',
   templateUrl: './dropdown.component.html',
   styleUrls: ['./dropdown.component.css']
 })
 export class DropdownComponent implements OnInit {
-isShowen = false;
-Text = '?מה התחביב שלך';
-
+  isShowen = false;
+  dropdownIcon = 'fa-solid fa-chevron-down';
+  @Input() dropdownLabel: string = "";
+  @Input() dropdownData: string[] = []; 
+  dropdownPlaceholder: string = '';
```

Comment:
not input?

Reply:
> not input?

I Assign its value inside the ngOnInit hook.